examples/notebooks/Helmholtz_non_periodic.ipynb

```diff
@@ -0,0 +1,326 @@
+     ]
+    }
+   ],
+   "source": [
+    "# Set the solver parameters\n",
+    "# 'cbig' -> Biconjugate gradient method\n",
+    "equation_h.set_solver('bicg', info=True, tol=1e-14)\n",
+    "\n",
+    "import time\n",
+    "\n",
+    "t0_s     = time.time()\n",
+    "uh, info = equation_h.solve()\n",
+    "t1_s     = time.time()\n",
+    "\n",
+    "\n",
+    "t0_d     = time.time()\n",
+    "l2_error = l2norm_h.assemble(u=uh)\n",
+    "h1_error = h1norm_h.assemble(u=uh)\n",
+    "t1_d     = time.time()\n",
+    "\n",
+    "print( '> CG info       :: ',info )\n",
+    "print( '> L2 error      :: {:.2e}'.format(l2_error))\n",
+    "print( '> H1 error      :: {:.2e}'.format(h1_error))\n",
+    "print( '> Solution time :: {:.2e}s'.format(t1_s - t0_s))\n",
+    "print( '> Evaluat. time :: {:.2e}s '.format(t1_d - t0_d))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 5: Saving the results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save the results using OutputManager\n",
+    "from psydac.api.postprocessing import OutputManager\n",
+    "import os\n",
+    "\n",
+    "os.makedirs('results_Helmholtz', exist_ok=True)\n",
+    "\n",
+    "Om = OutputManager(\n",
+    "    f'results_Helmholtz/space_info_{Omega.name}',\n",
+    "    f'results_Helmholtz/field_info_{Omega.name}',\n",
+    "# MPI version\n",
+    "#   comm=comm,    \n",
+    ")\n",
+    "\n",
+    "Om.add_spaces(V=Vh)\n",
+    "Om.export_space_info()\n",
+    "\n",
+    "Om.set_static()\n",
+    "Om.export_fields(u=uh)\n",
+    "\n",
+    "Om.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 6: Exporting the results to VTK and visualizing them with Paraview"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Export the results to VTK using PostProcessManager\n",
+    "from psydac.api.postprocessing import PostProcessManager\n",
+    "from sympy import lambdify\n",
+    "\n",
+    "Pm = PostProcessManager(\n",
+    "    domain=Omega,\n",
+    "    space_file=f'results_Helmholtz/space_info_{Omega.name}.yaml',\n",
+    "    fields_file=f'results_Helmholtz/field_info_{Omega.name}.h5',\n",
+    "# MPI version\n",
+    "#   comm=comm,    \n",
+    ")\n",
+    "\n",
+    "# The complex fields will be exported in two field: a real and an imaginary field\n",
+    "Pm.export_to_vtk(\n",
+    "    f'results_Helmholtz/visu_{Omega.name}',\n",
+    "    grid=None,\n",
+    "    npts_per_cell=3,\n",
+    "    fields='u',\n",
+    "    additional_physical_functions={'exact_solution': lambdify(Omega.coordinates, solution, modules='numpy')}\n",
+    ")\n",
+    "\n",
+    "Pm.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example: Results on the pretzel domain\n",
```

Comment:
This is a square pretzel :rofl:

Reply:
The famous one ^^'

Reply:
BTW this notebook has two empty cells at the bottom of the page. Please remove them :wink: